packages/icon/src/react/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { css } from 'glamor'
```

Comment:
Really wish we could get a workflow where I could see diffs compared to the last file instead of this situation where git thinks this is a brand new file. 

Currently:
I lose (most recent) git blame. It's way harder to review.